### PR TITLE
Add support to separate sync rgws from io rgws daemons in a multisite environment

### PR DESCRIPTION
**Pull request by mkasturi18**
# Description
This PR adds support to separate client IO daemons and sync rgw daemons.

Pass log - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-38YUOP/  

Post review _ pass logs - http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-F07E1V/ 

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by ckulal — COMMENTED** [review]
LGTM

File: suites/reef/rgw/tier-2_rgw_3_way_multisite.yaml [hunk 1]
```diff
@@ -258,10 +292,14 @@ tests:
...
-              - "ceph config set client.rgw.{daemon_id:shared.pri} rgw_realm india"
-              - "ceph config set client.rgw.{daemon_id:shared.pri} rgw_zonegroup shared"
-              - "ceph config set client.rgw.{daemon_id:shared.pri} rgw_zone primary"
-              - "ceph orch restart {service_name:shared.pri}"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.sync} rgw_realm india"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.sync} rgw_zonegroup shared"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.sync} rgw_zone primary"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.io} rgw_realm india"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.io} rgw_zonegroup shared"
+              - "ceph config set client.rgw.{daemon_id:shared.pri.io} rgw_zone primary"
```
Comment (hmaheswa) [inline suites/reef/rgw/tier-2_rgw_3_way_multisite.yaml:300]: @mkasturi18 , could you disable sync for io rgw's in the next PR as an enhancement

Comment (mkasturi18) [reply suites/reef/rgw/tier-2_rgw_3_way_multisite.yaml]: ack @hmaheswa , added the changes related to disabling sync on IO rgw daemons

File: suites/reef/rgw/tier-2_rgw_3_way_multisite.yaml [hunk 2]
```diff
@@ -271,10 +309,14 @@ tests:
...
-              - "ceph config set client.rgw.{daemon_id:shared.sec} rgw_zonegroup shared"
-              - "ceph config set client.rgw.{daemon_id:shared.sec} rgw_zone secondary"
-              - "ceph orch restart {service_name:shared.sec}"
+              - "ceph config set client.rgw.{daemon_id:shared.sec.sync} rgw_realm india"
+              - "ceph config set client.rgw.{daemon_id:shared.sec.sync} rgw_zonegroup shared"
+              - "ceph config set client.rgw.{daemon_id:shared.sec.sync} rgw_zone secondary"
+              - "ceph config set client.rgw.{daemon_id:shared.sec.io} rgw_realm india"
+              - "ceph config set client.rgw.{daemon_id:shared.sec.io} rgw_zonegroup shared"
+              - "ceph config set client.rgw.{daemon_id:shared.sec.io} rgw_zone secondary"
+              - "ceph orch restart {service_name:shared.sec.io}"
```
[the remark this answers is not in the document]
Comment (mkasturi18) [reply suites/reef/rgw/tier-2_rgw_3_way_multisite.yaml:351]: @TejasC88 , ack. Have added the changes.

**Review by hmaheswa — APPROVED** [review]

**Review by ckulal — APPROVED** [review]

**Pull request merged**